cmd/juju/status/output_tabular.go

```diff
@@ -71,11 +74,17 @@ func FormatTabular(writer io.Writer, forceColor bool, value interface{}) error {
 		header = append(header, "Notes")
 		values = append(values, message)
 	}
-
 	// The first set of headers don't use outputHeaders because it adds the blank line.
 	w := startSection(tw, true, header...)
-	w.Println(values...)
-
+	versionPos := indexOf("Version", header)
+	w.Print(values[:versionPos]...)
+	modelVersionNum, err := version.Parse(fs.Model.Version)
+	if staleVersion := (err == nil && jujuversion.Current.Compare(modelVersionNum) > 0); staleVersion {
```

Comment:
Are you sure that we need this var here?
Because there is no usage of it in further code.

Reply:
Corrected.